oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -1098,6 +1132,11 @@ public Builder setDefaultRetriesEnabled(boolean defaultRetriesEnabled) {
       return this;
     }
 
+    public Builder setUniverseDomain(String universeDomain) {
```

Comment:
Shouldn't this be provided by the superclass setUniverseDomain already?

Reply:
The base will return GoogleCredentials and user will have to cast it back to ServiceAccount